integration-tests/test/rpc.spec.ts

```diff
@@ -98,7 +97,7 @@ describe('Basic RPC tests', () => {
     it('should accept a transaction with a value', async () => {
       const tx = {
         ...defaultTransactionFactory(),
```

Comment:
Out of scope of this PR but I think `defaultTransactionFactory` should accept and object where the keys are the values that overrides what it returns by default